triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -114,7 +114,7 @@ object Server {
           // Start a new trigger given its identifier and the party it
           // should be running as.  Returns a UUID for the newly
           // started trigger.
-          path("start") {
+          path("v1" / "start") {
```

Comment:
Should we factor this out instead of duplicating it everywhere?

Reply:
I've done something in that direction. Have to say, a completely unreasonable amount of time was spent on working out something so simple 😢